### PR TITLE
Rename Sample App + Update Instructions

### PR DESCRIPTION
**Pull request by tamerbader**
- Renamed Example project from `MobilePaymentsSwiftUIExample` to `DonutCounter`
- Added instructions on how to utilize the MPSDK donut counter example app

### CODE REVIEW

File: Example/README.md [hunk 1]
```diff
@@ -1,17 +1,89 @@
-# Square Mobile Payments SDK Example App
```
Comment (brandonjenniges) [inline Example/README.md:1]: We may end up having multiple example apps so this README may need to be more generic but for now I think it's fine to have it specific to the donut counter app

File: Example/README.md [hunk 2]
```diff
@@ -1,17 +1,89 @@
...
+# 🍩 Donut Counter - A Square Mobile Payments SDK Example App
 
+<p align="center">
+<img src="../Images/donut-counter-home.png" width="300"/> <img src="../Images/donut-counter-take-payment.png" width="300"/>
+</p>
 
-### Assumptions and prerequisites
+Donut Counter is a sample application for building with the [Square Mobile Payments SDK](https://developer.squareup.com/docs/mobile-payments-sdk) on iOS. The app demonstrates how to install and initialize the SDK as well as utilizing the API's and user interfaces provided by the SDK to authorize a merchant, pair and manage card readers, and take a payment. To get started, follow the steps below.
```
Comment (brandonjenniges) [inline Example/README.md:7]: Maybe add something about it uses SwiftUI, or shows how to integrate into a SwiftUI app?

File: Example/README.md
Comment (brandonjenniges) [inline Example/README.md:1]: Love the updates to this! So much more detailed!

File: Scripts/templates/README.md.erb [hunk 3]
```diff
@@ -63,3 +63,7 @@ Follow the [Build on iOS guide](https://developer.squareup.com/docs/mobile-payme
 
 If you need more assistance, contact [Developer and App Marketplace Support](https://squareup.com/help/us/en/contact?panel=BF53A9C8EF68) or ask for help in the [Developer Forums](https://developer.squareup.com/forums/).
 
+## Mobile Payments SDK Example App
+<img src="./Images/donut-counter-home.png" width="300"/> <img src="./Images/donut-counter-take-payment.png" width="300"/>
+
+[Donut Counter](Example/) is a sample application included in the Examples folder of this repository, designed to help developers utilize the Square Mobile Payments SDK. This application serves as a practical guide for integrating and leveraging the SDK's capabilities within your own projects. It demonstrates essential functions such as initializing and authorizing the SDK, pairing with Square card readers and mock readers, and processing a $1.00 payment.
```
Comment (brandonjenniges) [inline Scripts/templates/README.md.erb:69]: Maybe add something about SwiftUI here too

File: Example/DonutCounter/DonutCounter/MobilePaymentsApp.swift
Comment (brandonjenniges) [inline Example/DonutCounter/DonutCounter/MobilePaymentsApp.swift:1]: nit: file name should probably change too

**Review by brandonjenniges — APPROVED** [review]

File: Example/README.md (hunk 2 above)
Comment (tamerbader) [reply Example/README.md]: Updated 🙏

File: Scripts/templates/README.md.erb (hunk 3 above)
Comment (tamerbader) [reply Scripts/templates/README.md.erb]: Updated 🙏

**Review by emma-squared — APPROVED** [review]

**Pull request merged**